backend/src/main/java/asegroup1/api/models/landregistry/LandRegistryQueryConstraint.java

```diff
@@ -13,7 +13,7 @@
  * 
```

Comment:
Lots of methods and attributes in this have too wide a scope according to my IDE, is this for testing?

Reply:
The only method i can see that being the case for are getRangeConstraint which is used for testing.